src/Data/Bin/Induction.agda

```diff
@@ -0,0 +1,76 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Arranging a well-founded recursion by _<_ on Bin.
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+module Data.Bin.Induction where
+
+open import Data.Bin.Base
+open import Data.Bin.Properties
+open import Data.Bin.Ordering
+open import Data.List using (List; []; _∷_; [_])
+open import Data.Nat as ℕ using (ℕ)
+import Induction.Nat
+open import Relation.Binary.PropositionalEquality using (subst)
+open import Relation.Nullary using (yes; no)
+open import Relation.Nullary.Negation using (contradiction)
+open import Induction.WellFounded using (Acc; module Inverse-image; module Subrelation)
+open Acc using (acc)
+
+Acc< =  Acc _<_
```

Comment:
I don't think any of these aliases help particularly. I think better to use the original names.

Reply:
The whole fragment with aliases is replaced with 
```
<-acc {x} = Subrelation.accessible {_} {_} {_} {Bin} {_<_} {_<ₙon_} <⇒<ₙon <ₙon-acc-x
  where
  -- Derive accessibility for _<_ on Bin from accessibility for _<_ on ℕ:
  <ₙon-acc-x = Inverse-image.accessible {_} {_} {_} {Bin} {ℕ} {ℕ._<_} toℕ
                                        (Induction.Nat.<-wellFounded (toℕ x))
```